frontend/src/pages/MapView/ResourceCard/styles.scss

```diff
@@ -160,6 +160,10 @@
     border-radius: 15px;
   }
 
+  .card-tag:hover {
+    background-color: #00000018;

```

Comment:
Is this valid a valid color hex? 😮

Reply:
yea its rgba lol, so i just made the rgb black and then made it very transparent with the a so that it was a light gray.